### ISSUE TITLE
Back/forward tlacidla v prehliadaci nefunguju 

### ISSUE THREAD

**Issue by eronisko** [marsher/volby-zo-zahranicia#22]
Skusil som v Chrome a Firefoxe

**Comment by filiplikavcan** [comment]
Mozem na to pozriet ak to este niekto nerobi.

**Comment by ondrek** [comment]
should be done — idem len testnut

**Comment by filiplikavcan** [comment]
Ako? Ja som to medzicasom fixol a trochu som aj zrefactoroval js, len este musim dokoncit refactoring navigacnych odkazov. Mam v tom pokracovat, ci uz je bug fixnuty?

**Comment by s7anley** [comment]
@filiplikavcan Najlepšie je separovať jednotlivé feature/bugfixy do rozličných PR, aby ti to nestálo zbytočne dlho lokálne. Teda na teraz by som odložil refaktoring odkazov a pripravil PR ak to máš hotové. Ak nie, kľudne sa do toho pustím ja.

**Comment by filiplikavcan** [comment]
Zajtra to dorobim a spravim PR.

**Comment by scholtz** [comment]
mozeme issue zavriet?

**Comment by eronisko** [comment]
U mna to funguje. Akurat ak chcem ist naspat z nahladu musim kliknut 2x (FF aj Chrome, Ubuntu)

**Comment by filiplikavcan** [comment]
Ano, pisem to aj v PR, ze to je issue. Dovodom je, ze robia prvy back na iframe a druhy na parent dokumente.

**Comment by scholtz** [comment]
hm.. mne to v chrome pod windowsom funguje.. mozno by sme to mohli pridat do otestovanych os/browser bug.. aku mas verziu os, ff a chrome?

vie to niekto otestovat pod macom?

**Comment by filiplikavcan** [comment]
Ja som to testoval na chrome na macu.

Sent from my phone

On Dec 14, 2015 9:02 PM, Ludovit Scholtz notifications@github.com wrote:

hm.. mne to v chrome pod windowsom funguje.. mozno by sme to mohli pridat do otestovanych os/browser bug.. aku mas verziu os, ff a chrome?

vie to niekto otestovat pod macom?

**Comment by eronisko** [comment]
Ubuntu 14.10, Chrome 46.0.2490.86, Firefox 42.0

**Comment by scholtz** [comment]
treba to opravovat? ja to neviem replikovat..

**Comment by filiplikavcan** [comment]
Ja sa na to mozem pozriet, ale asi az buduci tyzden. Podla mna to nie je kriticka vec a moze to tak zatial zostat.

**Comment by rlaffers** [comment]
Linux Mint 17.1
Chrome 46 - funguje bez problémov
Firefox 42 - problém presne ako píše @filiplikavcan. Prvý klik odchytí iframe. Po prvom kliku obsah iframe zmizne (celá jeho plocha sa vyšedí). Po druhom kliku už korektný návrat na predchádzajúcu obrazovku. V oboch prehliadačoch mám integrovaný pdf plugin.
